### ISSUE TITLE
Inconsistency of icons for stats when choosing a kart

### ISSUE THREAD

**Issue by Dinock** [supertuxkart/stk-code#4531]
## Description
When you are choosing a kart the icon for nitro efficiency differs in style from the other icons. The icon for the nitro is an image of a rendered 3d-model but the other three icons are drawn (see screenshot).

![STK-icons](https://user-images.githubusercontent.com/54438803/118122298-f4799b80-b3f2-11eb-8d05-8c7ad24ae764.png)

The icon for nitro efficiency should also be drawn to reach consistency in style for all icons.

## Steps to reproduce
Start a new game and have a closer look at the kart selection menu.

## Configuration

STK release version: Version 1.2

System: Ubuntu 21.04

**Comment by qwertychouskie** [comment]
The standard icon set has a lot of these types of inconsistencies, due to how long it's been around with various icons being added/copied as needed throughout the years without regards to whether the icon makes sense in the current location.  For example, this icon came from the Help screen, where a 3d model made slightly more sense there.

(FYI, the Cartoon skin shouldn't have these inconsistencies, as its icon set was created from scratch much more recently, therefore avoiding the whole pieced-together issue.)

**Comment by Benau** [comment]
Yes. I would recommend switching to "cartoon" skin if you would like a better interface, I have no intention to update the original icons anymore

Maybe after a while they will be removed anyway ![](https://cdn.lihkg.com/assets/faces/pig/sowhat.gif)

**Comment by Dinock** [comment]
Thanks for your answers and a suggestion:
Maybe in the next stable version of STK the Cartoon Skin could be set to the default skin. This would make the menus and interface look more consistent and polished right out of the box.

**Comment by CodingJellyfish** [comment]
![image](https://github.com/user-attachments/assets/e00c666c-d99d-4282-b1aa-d50f4da7ec8b)
